### DISCUSSION TITLE
This user account is blocked.

### DISCUSSION THREAD

**Discussion by bangaon** [vercel/community#4204] [Account Issue] [answered]
### Question

Just created an account on Vercel for the first time and seeing this. Please fix it.

![image](https://github.com/vercel/community/assets/145776365/578c33c9-78be-4eea-a952-aaa484d03db8)

**Comment by maktubz1** [comment]
I can't able to signup with my personal account, I'm not doing any business on my account.

**Comment by vijethasoftwares** [comment]
Same with me! this is annoying.

**Comment by hungchihsueh** [comment]
I can't able to signup with my personal account, I'm not doing any business on my account.!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

**Comment by amyegan** [comment, marked as answer]
Sorry you're having trouble signing up for a new account. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps. 

For more info, please check this guide: [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)
